server/reports/encounters/2_6/convert_data_js/src/convert_data.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'main' {
+    // Extism exports take no params and return an I32
+    export function convert_data(): I32;
+  }
+  declare module 'extism:host' {
+    interface user {
+      sql(ptr: I64): I64;
```

Comment:
I get red squigglies on these types `I64` and `I32`, do I need to do something to the type config?

Reply:
I think there is a setup for the new reports thing that I am missing and not familiar with - could you point me at some docs if they exist?

Reply:
https://github.com/msupply-foundation/open-msupply/blob/develop/server/report_builder/README.md

Reply:
Huh my editors says they are unresolved but doesn't complain about it 👀 

I'm actually not 100% sure how this d.ts file ends up mapping to anything, I copied from the other examples, assume it helps extism figure something out?? 
![Screenshot 2025-02-05 at 11 26 26 AM](https://github.com/user-attachments/assets/a34532d7-a49e-4713-95f8-fc1270f08e2a)